src/formats/mmCIF.cpp

```diff
@@ -159,22 +157,14 @@ void mmCIFFormat::init_() {
     steps_positions_.push_back(position);
 
     if (atom_site_map_.find("type_symbol") == atom_site_map_.end()) {
-        throw format_error("CIF files does not define type symbol");
+        throw format_error("could not find _atom_site.type_symbol in '{}'", file_.path());
     }
 
-    // Attempt to guess how coordinates are stored
-    auto cartn_x = atom_site_map_.find("cartn_x");
-    auto fract_x = atom_site_map_.find("fract_x");
-    if (cartn_x != atom_site_map_.end()) {
-        uses_fract_ = false;
-    } else if (fract_x != atom_site_map_.end()) {
-        uses_fract_ = true;
-    } else {
-        throw format_error("CIF file does not define coordinates");
+    if (atom_site_map_.find("Cartn_x") == atom_site_map_.end()) {
```

Comment:
Okay, so this code no longer supports fractional coordinates?

Reply:
Exactly. Support for fractional coordinates is now done with GEMMI/the CIF reader introduced in #356.